django/contrib/gis/db/models/fields.py

```diff
@@ -280,6 +255,15 @@ def formfield(self, **kwargs):
             defaults['widget'] = forms.Textarea
         return super().formfield(**defaults)
 
+    def select_format(self, compiler, sql, params):
+        """
+        Return the selection format string, depending on the requirements
+        of the spatial backend.  For example, Oracle and MySQL require custom
+        selection formats in order to retrieve geometries in OGC WKB.
+        """
+        select = compiler.connection.ops.select
```

Comment:
Maybe defining `select_operation = 'select'` on the class and overriding it to `select_extend` in `ExtendField`, along with `getattr(compiler.connection.ops, self.select_operation)` here could eliminate the duplication in `ExtendField.select_format()`?